package.json

```diff
@@ -12,19 +12,19 @@
     "lint": "yarn prettier . --ignore-unknown --check"
   },
   "dependencies": {
-    "expect": "^27.5.1",
-    "jest-circus": "^27.5.1",
-    "jest-each": "^27.5.1",
-    "jest-mock": "^27.5.1",
-    "jest-snapshot": "^27.5.1",
+    "expect": "^28.0.0",
+    "jest-circus": "^28.0.0",
+    "jest-each": "^28.0.0",
+    "jest-mock": "^28.0.0",
+    "jest-snapshot": "^28.0.0",
     "piscina": "^3.1.0",
     "supports-color": "^9.2.1"
   },
   "devDependencies": {
     "prettier": "^2.6.2"
   },
   "peerDependencies": {
-    "jest": "^27.0.0"
+    "jest": "^28.0.0"
```

Comment:
I'd assume it still works fine with both versions, if you wanna do `^27.0.0 || ^28.0.0`